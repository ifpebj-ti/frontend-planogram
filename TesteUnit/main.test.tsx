
import React from 'react'; 
import { render } from '@testing-library/react';
import Home from '../app/page';

it('should render the main component', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});

