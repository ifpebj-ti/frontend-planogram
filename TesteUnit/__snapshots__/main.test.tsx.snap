// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the main component 1`] = `
<DocumentFragment>
  <div>
    <div
      class="sidenav "
    >
      <button
        class="toggle-button"
      >
        <img
          alt="Menu Toggle Icon"
          height="70%"
          src="/assets/images/logo2.png"
          width="90%"
        />
      </button>
      <div
        class="menu-content"
      >
        <div
          class="menu-item"
        >
          <a
            href="/"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 576 512"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
              />
            </svg>
          </a>
        </div>
        <div
          class="menu-item"
        >
          <a
            href="/importar"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M16 288c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h112v-64zm489-183L407.1 7c-4.5-4.5-10.6-7-17-7H384v128h128v-6.1c0-6.3-2.5-12.4-7-16.9zm-153 31V0H152c-13.3 0-24 10.7-24 24v264h128v-65.2c0-14.3 17.3-21.4 27.4-11.3L379 308c6.6 6.7 6.6 17.4 0 24l-95.7 96.4c-10.1 10.1-27.4 3-27.4-11.3V352H128v136c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H376c-13.2 0-24-10.8-24-24z"
              />
            </svg>
          </a>
        </div>
        <div
          class="menu-item"
        >
          <a
            href="/configuracoes"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
              />
            </svg>
          </a>
        </div>
        <div
          class="menu-item"
        >
          <a
            href="/sair"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M497 273L329 441c-15 15-41 4.5-41-17v-96H152c-13.3 0-24-10.7-24-24v-96c0-13.3 10.7-24 24-24h136V88c0-21.4 25.9-32 41-17l168 168c9.3 9.4 9.3 24.6 0 34zM192 436v-40c0-6.6-5.4-12-12-12H96c-17.7 0-32-14.3-32-32V160c0-17.7 14.3-32 32-32h84c6.6 0 12-5.4 12-12V76c0-6.6-5.4-12-12-12H96c-53 0-96 43-96 96v192c0 53 43 96 96 96h84c6.6 0 12-5.4 12-12z"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
    <div
      class="shelf-selector"
    >
      <h1>
        Selecione a prateleira que deseja visualizar
      </h1>
      <div
        class="shelf-container"
      >
        <div
          class="shelf-section"
        >
          <a
            href="/prateleira/1"
          >
            <button
              class="shelf-number"
            >
              Prateleira 1
            </button>
          </a>
          <div
            class="shelf-items"
          >
            <a
              href="/prateleira/1"
            >
              <button
                class="shelf-item"
              >
                Prateleira 
              </button>
            </a>
            <a
              href="/prateleira/1"
            >
              <button
                class="shelf-item"
              >
                Prateleira 
              </button>
            </a>
          </div>
          <a
            href="/prateleira/1"
          >
            <button
              class="shelf-number"
            >
              Prateleira 1
            </button>
          </a>
        </div>
        <div
          class="shelf-section"
        >
          <a
            href="/prateleira/2"
          >
            <button
              class="shelf-number"
            >
              Prateleira 2
            </button>
          </a>
          <div
            class="shelf-items"
          >
            <a
              href="/prateleira/2"
            >
              <button
                class="shelf-item"
              >
                Prateleira 
              </button>
            </a>
            <a
              href="/prateleira/2"
            >
              <button
                class="shelf-item"
              >
                Prateleira 
              </button>
            </a>
          </div>
          <a
            href="/prateleira/2"
          >
            <button
              class="shelf-number"
            >
              Prateleira 2
            </button>
          </a>
        </div>
        <div
          class="shelf-section"
        >
          <a
            href="/prateleira/3"
          >
            <button
              class="shelf-number"
            >
              Prateleira 3
            </button>
          </a>
          <div
            class="shelf-items"
          >
            <a
              href="/prateleira/3"
            >
              <button
                class="shelf-item"
              >
                Prateleira 
              </button>
            </a>
            <a
              href="/prateleira/3"
            >
              <button
                class="shelf-item"
              >
                Prateleira 
              </button>
            </a>
          </div>
          <a
            href="/prateleira/3"
          >
            <button
              class="shelf-number"
            >
              Prateleira 3
            </button>
          </a>
        </div>
      </div>
      <footer
        class="footer"
      >
        <p>
          Todos os direitos reservados
        </p>
      </footer>
    </div>
  </div>
</DocumentFragment>
`;
